CMakeLists.txt

```diff
@@ -274,17 +274,11 @@ set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}")
 endif()
 
 ### ---[ Find universal dependencies
-find_package(OpenMP COMPONENTS C CXX)
-if(OpenMP_FOUND)
+find_package(OpenMP)
+if(OPENMP_FOUND)
```

Comment:
Why changing this?

Reply:
Okay, this is a Github bug.

If you click on the commit `fix merge conflict` (4563d59), it redirects [here](https://github.com/PointCloudLibrary/pcl/pull/3814/commits/4563d59ceb012516952d72d78b88151326375850), on which there are **240 files changed** (oh boy!)

Although if you click on the hash 4563d59, you can see there are only 3 files changed.

Also, here is my local for that commit:
```
(fix/add_to_clang_tracking)→ git show --name-only 4563d59   
commit 4563d59ceb012516952d72d78b88151326375850
Merge: 332294ef9 8d7033e24
Author: Divyanshu Madan <iamdm99@gmail.com>
Date:   Sat Apr 4 13:43:48 2020 +0530

    fix merge conflict

tracking/include/pcl/tracking/impl/kld_adaptive_particle_filter_omp.hpp
tracking/include/pcl/tracking/impl/particle_filter_omp.hpp
tracking/include/pcl/tracking/impl/pyramidal_klt.hpp

```

It is still showing unresolved merge conflicts, which my branch has nothing to do with. I fixed that and had to reset that change after observing this behaviour.
I hope this gets rectified soon :laughing: 

So, don't worry about that, I didn't change that. (At least I hope :laughing: )